src/utils.ts

```diff
@@ -27,3 +29,20 @@ export async function randomArray(size: number): Promise<string> {
         });
     });
 }
+
+type JsonValue = null | string | number;
+type JsonArray = Array<JsonValue | JsonObject | JsonArray>;
+interface JsonObject {
+    [key: string]: JsonObject | JsonArray | JsonValue;
+}
+type Json = JsonArray | JsonObject;
+
+/**
+ * Utility to load JSON file from the local filesystem synchronously.
+ * Preferred over `require` due to how it may execute javascript code if the file does not exist.
+ * @param paths takes an array of paths which will be joined using the system's path delimiter
```

Comment:
I don't think it's correct that it takes an array of paths.

```suggestion
  * @param paths - An array of path segments which will be joined using the system's path delimiter.
```

Reply:
It is to avoid the cross-platform footgun that is Windows path backslashes

Reply:
I know what it's for, I just want the comment not to be full of lies.